app/assets/stylesheets/custom.css.scss

```diff
@@ -90,6 +92,7 @@ h1 {
 }
 
 #flash_errors{
+  font-family: monospace;
```

Comment:
Properties should be ordered background-color, border-radius, color, font-family, margin-right, opacity, padding, text-align, text-color, text-style